client/components/Main.jsx

```diff
@@ -1,18 +1,30 @@
 import React, {Component} from 'react'
 import {connect} from 'react-redux'
 import {fetchProducts} from '../store.js'
-import {BrowserRouter, Route} from 'react-router-dom'
+import {BrowserRouter, Route, NavLink} from 'react-router-dom'
 import ProductList from './ProductList'
 import Product from './Product'
+import Cart from './Cart';
 
 class Main extends Component {
 
 	render(){
 		return (
 			<div className='container'>
 				<h1> HARQ Store </h1>
+					<div className="container pull-right">
+							<ul className="nav navbar-nav">
+								<li>
+									<NavLink to="/" activeClassName="active">Home</NavLink>
+								</li>
+								<li>
+									<NavLink to="/yourCart" activeClassName="active">Your Cart</NavLink>
+								</li>
+							</ul>
+					</div>
 				<Route exact path='/' component={ProductList} />
 				<Route exact path='/products/:productId' component={Product} />
+				<Route exact path='/yourCart' component={ Cart } />
```

Comment:
Although it makes sense in Amazon, a RESTful app has to be REST the whole way. So this has to be /order/orderId.

BUT: I think we can keep it static on the page for now. so just <Cart />

Reply:
I agree with Qian, but I also like the /yourCart.
Implementing Cart as active order is just our implementation detail. No need to expose that to the users in the Url. Kinda like we hide our implementation detail with /yourCart.
I guess I am ok either way. lol